packages/RNTester/.detoxrc.json

```diff
@@ -4,15 +4,15 @@
   "configurations": {
     "android.emu.debug": {
       "binaryPath": "android/app/build/outputs/apk/debug/app-debug.apk",
-      "build": "cd android && ./gradlew assembleDebug assembleAndroidTest -DtestBuildType=debug && cd ..",
+      "build": "cd android && ./gradlew assembleDebug app:assembleAndroidTest -DtestBuildType=debug && cd ..",
       "type": "android.emulator",
       "device": {
-        "avdName": "testAVD"
+        "avdName": "Pixel_API_28"
       }
     },
     "android.emu.release": {
       "binaryPath": "android/app/build/outputs/apk/release/app-release.apk",
-      "build": "cd android && ./gradlew assembleRelease assembleAndroidTest -DtestBuildType=release && cd ..",
+      "build": "cd android && ./gradlew assembleRelease app:assembleAndroidTest -DtestBuildType=release && cd ..",
       "type": "android.emulator",
```

Comment:
these changes have already been merged, will it create any issues while merging?

Reply:
@YashKumarVerma i don't think so they should